blog/2015/03/and-now-we-are-three.html

```diff
@@ -0,0 +1,72 @@
+---
+layout: blog
+root: ../../..
+author: Greg Wilson
+title: "And Now We Are Three"
+date: 2015-03-15
+time: "06:00:00"
+category: ["Teaching"]
+---
+<!-- start excerpt -->
+<p>
+  The four core topics that every Software Carpentry workshop is supposed to teach are
+  automating tasks using the Unix shell,
+  structured programming in Python, R, or MATLAB,
+  version control using Git or Mercurial,
+  and data management using SQL.
+  In practice,
+  many workshops omit the fourth,
+  either because they want to put more time into the first three,
+  or because the instructors and learners don't think most scientists need to know it.
```

Comment:
I wonder if there's a way to rephrase this so it's not putting words in the mouth of instructors. Something like "or because the instructors want to teach a topic they consider more important to their students."